BAC0/core/io/Read.py

```diff
@@ -84,13 +85,13 @@ class ReadProperty:
 
     def read(
         self,
-        args,
-        arr_index=None,
-        vendor_id=0,
+        args: str,
+        arr_index: t.Optional[int] = None,
+        vendor_id: int = 0,
         bacoid=None,
-        timeout=10,
-        show_property_name=False,
-    ):
+        timeout: int = 10,
+        show_property_name: bool = False,
+    ) -> t.Union[ReadValue, t.Tuple[ReadValue, str], None]:
```

Comment:
it seems like `read` can sometimes also return lists? like when we read a priority array?